cabal.project

```diff
@@ -1 +1,4 @@
 packages: .
+
+if arch(javascript)
+  extra-packages: ghci
```

Comment:
eh?

Reply:
That's for `haskell.nix` (or maybe even `cabal` in general).

Reply:
That's sort of fucked up! But at least it is just in `cabal.project`.